### PR TITLE
[Merged by Bors] - chore(IsAlgClosed): move some defs about lift into IsAlgClosed namespace

### PR DESCRIPTION
**Pull request by ChrisHughes24**
---
All of the moved theorems are basically auxiliary things which need not be used by users anyway.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/FieldTheory/IsAlgClosed/Basic.lean [hunk 1]
```diff
@@ -4,7 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kenny Lau
 -/
 import Mathlib.FieldTheory.Normal
-import Mathlib.FieldTheory.Perfect
+import Mathlib.FieldTheory.PerfectClosure
```
Comment (ocfnash) [inline Mathlib/FieldTheory/IsAlgClosed/Basic.lean:7]: Why is the import changing?

Comment (ChrisHughes24) [reply Mathlib/FieldTheory/IsAlgClosed/Basic.lean:7]: Entirely by accident. I'll fix it

**Comment by ocfnash** [comment]
Once the import change is reverted this is good to go, thanks!

bors d+

File: Mathlib/FieldTheory/IsAlgClosed/Basic.lean (hunk 1 above)
Comment (ChrisHughes24) [reply Mathlib/FieldTheory/IsAlgClosed/Basic.lean:7]: Entirely by accident. I'll fix it

**Pull request closed without merging**